features/auto_mtls.md

```diff
@@ -168,13 +164,13 @@ Not Applied.
 
 **Bugs**
 
-- [ ] Feature has no known major issues.
+- [X] Feature has no known major issues.
 
 **Approvals**
 
-- [ ] The appropriate work group(s) have reviewed and approved promotion of the feature.
-- [ ] The supportability review panel has reviewed promotion of the feature.  
-- [ ] The TOC has reviewed and approved promotion of the feature as part of the
+- [X] The appropriate work group(s) have reviewed and approved promotion of the [feature](https://docs.google.com/document/d/1cU4TFqYdir1luUN4gf3mTvujb4LugNLhcNffOluQyns/edit?disco=AAAALq26fXg&ts=60467f2f&usp_dm=true&resourcekey=0-tIc8kq3LHGokG4FHfXJ5CQ).
+- [ ] The supportability review panel has reviewed promotion of the feature. TODO: WIP.  
+- [X] The TOC has reviewed and approved promotion of the feature as part of the
```

Comment:
You can also refer to Istio 2021 roadmap (https://docs.google.com/spreadsheets/d/1FJ3AJFFnGw6TxZAoyj-zgFwTn37WsMKTOGNR4FjG7T8/edit#gid=117171340) which has been reviewed and approved by TOC.